recipes/sssp/meta.yaml

```diff
@@ -0,0 +1,212 @@
+{% set name = "sssp" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - fn: pbe-efficiency.tar.gz
+    url: https://archive.materialscloud.org/record/file?record_id=862&filename=SSSP_{{ version }}_PBE_efficiency.tar.gz
+    sha256: ef055105b6d84f3c7cd4ae0f99b5db58418a687a68c9d83c19a3d2f2cbf89727
+    folder: efficiency
+  - fn: pbe-precision.tar.gz
+    url: https://archive.materialscloud.org/record/file?record_id=862&filename=SSSP_{{ version }}_PBE_precision.tar.gz
+    sha256: 93296be647a6ecb3331b7a1763c7eed6fd5cf5c597ee27954da6e1fe13c05d95
+    folder: precision
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
+
+test:
+  commands:
+    - test -f ${PREFIX}/share/sssp/efficiency/Ag_ONCV_PBE-1.0.oncvpsp.upf  # [unix]
```

Comment:
We can do that in the feedstock but can we use a glob or something instead of a list like this?